docs/styleguide/PyGuide.md

```diff
@@ -816,3 +816,18 @@ This is required so that the autogenerated API documentation is rendered properl
 
 If the autogenerated API documentation does not show type hints for certain arguments despite the fact that the type hints are present in the object's implementation code,
 or if the type hints do not refer to the API symbol's canonical alias, then the type hint should be explicitly declared in the docstring using the `:type *param_name*:` directive (or `:rtype:` for return types).
+
+<a id="s5-test-suite-coding-rules"></a>
+<a id="5-test-suite-coding-rules"></a>
+<a id="test-suite-coding-rules"></a>
+
+## 5 Test suite coding rules
```

Comment:
Should be index 6. 
And link to "Table of Contents"

Reply:
I would expect this to be caught by markdownlint. Thanks for spotting.

Reply:
unfortunately markdownlint doesn't check index in headers